packages/react-components/react-color-picker-preview/library/docs/Spec.md

```diff
@@ -2,62 +2,458 @@
 
 ## Background
 
-_Description and use cases of this component_
+The ColorPicker is used to browse through and select colors.
+By default, it lets people navigate through colors on a color spectrum; or specify a color in either Red-Green-Blue (RGB); or alpha color code; or Hexadecimal textboxes.
 
 ## Prior Art
 
-_Include background research done for this component_
+- [Convergence epic](https://github.com/microsoft/fluentui/issues/31778)
 
-- _Link to Open UI research_
-- _Link to comparison of v7 and v0_
-- _Link to GitHub epic issue for the converged component_
+### Fabric (v8)
+
+```jsx
+import * as React from 'react';
+import {
+  ColorPicker,
+  ChoiceGroup,
+  IChoiceGroupOption,
+  getColorFromString,
+  IColor,
+  IColorPickerStyles,
+  IColorPickerProps,
+} from '@fluentui/react';
+
+const white = getColorFromString('#ffffff')!;
+
+const ColorPickerBasicExample: React.FunctionComponent = () => {
+  const [color, setColor] = React.useState(white);
+  const [showPreview, setShowPreview] = React.useState(true);
+  const [alphaType, setAlphaType] = React.useState<IColorPickerProps['alphaType']>('alpha');
+
+  const updateColor = React.useCallback((ev: any, colorObj: IColor) => setColor(colorObj), []);
+
+  return (
+    <>
+      <ColorPicker
+        color={color}
+        onChange={updateColor}
+        alphaType={alphaType}
+        showPreview={showPreview}
+        styles={colorPickerStyles}
+        strings={{
+          hueAriaLabel: 'Hue',
+        }}
+      />
+    </>
+  );
+};
+```
+
+### 3rd party Design Systems
+
+- Adobe Spectrum
+  - [ColorPicker](https://react-spectrum.adobe.com/react-spectrum/ColorPicker.html)
+  - [ColorArea](https://react-spectrum.adobe.com/react-spectrum/ColorArea.html)
+  - [ColorSlider](https://react-spectrum.adobe.com/react-spectrum/ColorSlider.html)
+
+### Components
+
+| Purpose                                                                  | Fabric (V8)    | V9          | Matching? |
+| ------------------------------------------------------------------------ | -------------- | ----------- | --------- |
+| Component responsible for color editing using ColorArea and ColorSliders | ColorPicker    | ColorPicker | ⚠️        |
+| ColorArea allows user to pick a color using two channels                 | ColorRectangle | ColorArea   | ⚠️        |
+| ColorSlider allows user to pick a color using individual channel         | ColorSlider    | ColorSlider | ⚠️        |
+| AlphaSlider allows user to pick a color using alpha channel              |                | AlphaSlider | ❌        |
 
 ## Sample Code
 
-_Provide some representative example code that uses the proposed API for the component_
```

Comment:
without this line "Sample Code" is a pretty generic term. i wouldn't know if this code below already existed or what. Would be helpful to add a comment about 'proposed API below'